libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp

```diff
@@ -617,16 +617,16 @@ uint32_t AP_Frsky_Telem::calc_gps_latlng(bool *send_latitude)
     // alternate between latitude and longitude
     if ((*send_latitude) == true) {
         if (loc.lat < 0) {
-            latlng = ((abs(loc.lat)/100)*6) | 0x40000000;
+            latlng = ((labs(loc.lat)/100)*6) | 0x40000000;
```

Comment:
I don't understand this change, loc.lat is int32_t, so abs will return an int . What is the problem here ? 
Is the problem that we try to put an int32_t in and uint32_t ? in that case, isn't a cast better ?

Reply:
I don't know why but with abs() I got compilation error "float in logical operator"

Reply:
ok, what compiler are you using ? gcc-4.9 ?

Reply:
inspite of compiler, tested from 4.8 to 5.4

Reply:
that is odd, it compile fine on my side ... but with WAF ... I will try to understand why it failed later

Reply:
seems that somehow GCC uses float abs() instead auto one